com.github.sdbg.debug.ui/src/com/github/sdbg/ui/launchers/IChromeLaunchShortcutExt.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sdbg.ui.launchers;
+
+import org.eclipse.core.resources.IResource;
+
+public interface IChromeLaunchShortcutExt {
```

Comment:
I don't think this would be extending the launching system.  Rather it would be reversing how launch options are added to this menu.  Instead of Gwt saying, "I know about SDBG and will include it as a dependency and create a place for it", Gwt would be saying, "I provide an extension point where any plugin can tell me about an item they want to add to the launch menu".